Week1/assets/New folder/App.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
```

Comment:
I see that you have to separate your files yes But you should at least mention that which files are for what !!

As I saw it's same as the one in the assets folder...